persistence/persistence-mybatis/src/main/java/org/cbioportal/persistence/mybatis/util/CacheMapUtil.java

```diff
@@ -89,11 +89,17 @@
     }
 
     @PostConstruct
-    private void initializeCacheMemory() {
+    private void init() {
+        initializeCacheMemory();
+    }
+
+    public void initializeCacheMemory() {
```

Comment:
I think this should be synchronized, especially since these maps aren't thread safe.
```suggestion
    public synchronized void initializeCacheMemory() {
```

Reply:
Done